.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,23 +3,5 @@
 Fixes (issue #)
 
 <!--
-Please include a summary of what you want to achieve in this pull request. Remember to indicate the affected package(s).
+Please include a summary of what you want to achieve in this pull request. Remember to add a changeset that indicates the affected package(s) and if they are major / minor / patch changes by using `yarn changeset`.
```

Comment:
`add a changeset`
I'd suggest link-ifying this section of text and have it point to the section in our release guidelines that talks about changesets. It'll be new tech to a bunch of folks, and knowing how developers can get nerd-sniped I'd expect that a bunch will bail out of the sentence at this point to go read about what changesets are. Linking them will help us control where they find that info instead of starting with the public docs on changesets.